extensions/amp-list/0.1/test/test-amp-list.js

```diff
@@ -1121,7 +1121,7 @@ describes.repeated(
               element.setAttribute('binding', 'refresh');
             });
 
-            it('should rescan() with {update: false} before FIRST_MUTATE', async () => {
+            it('should rescan() with {update: false, evaluate: false} before FIRST_MUTATE', async () => {
```

Comment:
I thought `evaluate` is not an option for rescan? Was there any consideration to introduce this property re: [typing](https://github.com/ampproject/amphtml/pull/27876/files#r412481801)?

Reply:
Oops, another out of date comment.